lib/bootstrap_form/inputs/text_field.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
```

Comment:
Do you expect there will be one file for each of the FIELD_HELPERS? Did you consider using one file and one method to define the "typical" helpers listed in FIELD_HELPER? The current implementation makes it clear that the intent is that all those helpers are basically the same, but it's harder to see because of the Ruby meta-programming.

Reply:
So my short term intent is to get it working. My shorter term intent is to either subclass or use metaprogramming. I'm leaning towards subclassing though. It feels cleaner and easier to read. we can use class-level attribute to determine the field name `self.field_name = :text_field` for instance.

Reply:
I'm not sure where what you mean. If you don't mind the risk that we might decide not to go in that direction, why don't you continue working on this PR, so I get a better idea of where you're going. 

Did you have any ideas about what to do with `form_group` and `form_group_builder`, and the methods below them?

Reply:
@lcreid to answer your questions about form_group/form_group_builder, I think this is a problem best tackled separately.